### ISSUE TITLE
503 When browsing from the web

### ISSUE THREAD

**Issue by astraldragon** [jwilder/nginx-proxy#978]
Hi,

I'm fairly new to docker and to setting up servers so bear with me.  Basically what I'm trying to solve is I have a decent amount of apps (~10) and I'd like to give them nice urls for my own sanity.  I only access this server from within the local network so I don't really care about opening it up outside of that.  I made a smaller scope set up just to try to get it working the way I want and I can't seem to figure it out.  Everything fires up and I can curl the portainer url but if I try to access it via a url I get a 503.  I think I'm misunderstanding what it sets up for URLs.  Here is my config.

```yaml
# /opt/docker-compose.yml
    
---

version: '3.4'
services:
  nginx-proxy:
    image: jwilder/nginx-proxy
    container_name: nginx-proxy
    ports:
      - 81:80
    volumes:
      - /var/run/docker.sock:/tmp/docker.sock:ro
  portainer:
    image: portainer/portainer
    container_name: portainer
    restart: unless-stopped
    ports:
      - 9000:9000
    volumes:
      - /opt/appdata/portainer:/data
      - /var/run/docker.sock:/var/run/docker.sock
    environment:
      - VIRTUAL_HOST=portainer
```

And the ngnix config:

```yaml
server {
        server_name _; # This is just an invalid value which will never trigger on a real hostname.
        listen 80;
        access_log /var/log/nginx/access.log vhost;
        return 503;
}
# portainer
upstream portainer {
                                ## Can be connect with "opt_default" network
                        # portainer
                        server 172.18.0.2:9000;
}
server {
        server_name portainer;
        listen 80 ;
        access_log /var/log/nginx/access.log vhost;
        location / {
                proxy_pass http://portainer;
        }
}
```

So basically what I have right now for a url is:

```
192.168.1.10:9000
```
And I want 
```
192.168.1.10/portainer
```

I know that I don't have to map the port if there is only one being exported, but I still have it set up just because I still use this server and still want to access it by port while trying to figure this set up out.  Is what I want not really what this container was meant for?  Is there another container I could use if that is the case?

**Comment by DeanMW** [comment]
@astraldragon Do you have any update on this? Im running into the same issue right now with a very similar config. I have a feeling this might not be working correct with swarm.
